Pilot/pilotCommands.py

```diff
@@ -484,27 +474,21 @@ def execute(self):
 
     # If NumberOfProcessors or MaxRAM are defined in the resource configuration, these
     # values are preferred
+
+    # pilotProcessors is basically the number of processors this pilot is "managing"
+    self.pp.pilotProcessors = numberOfProcessorsOnWN
+
+    # payloadProcessors is the number of processors used by the single payloads.
+    # We store payloadProcessors in the global parameters so that other commands can more easily use it.
+    # (MultiLaunchAgent is right now is the only consumer)
+    self.pp.payloadProcessors = 1
     if "WholeNode" in self.pp.tags:
-      self.pp.payloadProcessors = numberOfProcessorsOnWN
-    if self.pp.maxNumberOfProcessors > 0:
```

Comment:
This allows payloadProcessors to be > 1 and is used to make the multiprocessor payloads work.